packages/editor/src/components/widgets/code-editor/index.ts

```diff
@@ -1 +1,4 @@
 export { default as CodeEditor } from './CodeEditor';
+export { default as ResizableCodeEditor } from './ResizableCodeEditor';
+export { default as SingleLineCodeEditor } from './SingleLineCodeEditor';
```

Comment:
We can keep this name and it's very intuitive, but still a small remark.
In the process.model apis and bpm.exec we tried to distinguish single line codes from long statements by calling single liners "expression" (ScriptExpression, MacroExpression) and so on. 
If you like the term, you may adapt it here to, and name it "CodeExpressionEditor"?